team/Product-ecosystem.md

```diff
@@ -4,32 +4,33 @@ Current hosting: Contigex and AWS.
 
 mostly Rails apps and one NodeJS app.
 
-* prx.org (Rails 2)
+* exchange.prx.org (Rails 2)
 * count.prx.org (analytics for listening and viewing assets) (NodeJS)
 * castle.prx.org (mysql, star schema, data mart: big database with API, data only for prx.org) (Rails 3)
```

Comment:
good question. IIRC there is a [legacy castle app at Contegix](https://github.com/PRX/TheCastle) and the ["actual" production castle in Elixir](https://github.com/PRX/castle.prx.org). This one is the legacy app.

@kookster may be able to clarify.

Reply:
yeah, this is the "legacy" castle on contegix.  which does still own the `castle.prx.org` domain (the next-gen elixir castle is currently at `castle.prx.tech`).

I would leave the legacy "TheCastle" app here, since it works hand-in-hand with exchange and count.

The elixir castle app I'd group with our newer HAL backend services ... cms.prx.org and feeder.prx.org.